owlbot.py

```diff
@@ -129,13 +129,13 @@
     s.replace(
         ".kokoro/samples/python3.*/common.cfg",
         """env_vars: \{
-        key: "BUILD_SPECIFIC_GCLOUD_PROJECT"
```

Comment:
This added indentation seems to be the cause of common.cfg not corrected modified by owlbot.py.